### ISSUE TITLE
How to reduce the size of best.pt

### ISSUE THREAD

**Issue by suigong1** [ultralytics/yolov5#13034]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

I finished a simple train with 60 images and 1 tag, but after 100 training epochs, it generated a best.pt file with 14MB, what can I do to reduce the size of the best.pt, like changing the epochs or pre-training weights or batch-size?
Looking forward to your further reply!

**Comment by glenn-jocher** [comment]
@suigong1 hey there! 🚀 Reducing the size of your `best.pt` model file can be approached in a few ways:

1. **Model Selection**: Opt for a smaller model architecture like YOLOv5s if you haven't already. This has fewer parameters and thus will produce a smaller model file.

   ```bash
   python train.py --data custom.yaml --weights yolov5s.pt
   ```

2. **Pruning**: After training, you can apply model pruning techniques to remove less important weights, which can reduce the model size significantly.

3. **Quantization**: Implementing quantization can also reduce the model size by decreasing the precision of the weights.

Keep in mind that reducing the model size might affect the accuracy and robustness of your model. If you need more detailed guidance, check out our tips for best training results at https://docs.ultralytics.com/guides/model-training-tips//. Good luck! 🍀
